testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/AIALoginPage.java

```diff
@@ -0,0 +1,21 @@
+package org.keycloak.testsuite.ui.account2.page;
+
+import java.net.URI;
+import java.util.stream.Stream;
+
+import org.keycloak.testsuite.auth.page.login.OIDCLogin;
+
+public class AIALoginPage extends OIDCLogin {
```

Comment:
Do we really need this page object? Isn't it enough to use generic login page?

Reply:
looks the generic page is enough, removing